### PR TITLE
Finished the 'News' Page

### PR DESCRIPTION
**Pull request by bghsstemclub**
I created an 'Article_holder' page as well as 2 article pages. I also finished the automatic slide show. The reason there are changes in all of the files is because I added a 'News' option and deleted the 'Game Designs' one in the menu list.

### CODE REVIEW

File: 3D-designs.html [hunk 1]
```diff
@@ -1,51 +1,52 @@
...
     </div>
-    <div class="vertical_line"></div>
-    <div class="center2"><b>3D Designs</b></div>
-    <div class="center"><img style="width:250px" src="STEM Pictures/STEM logo1.png" alt="STEM_Logo"></div>
-  </div>
-  <!--End of Menu Section-->
+    <!--End of Menu Section-->
 
-  <h3>Our Weekly Icebreakers</h3>
+    <h3>Our Weekly Icebreakers</h3>
```
Comment (Teamustu) [inline 3D-designs.html:50]: Why is this in the 3d-design page?

File: news.css [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+.crop img{
+    width: 100%;
+    height: auto;
+}
+
+/*fade when hover over (for the button)*/
+#fade3{
+    position: absolute; 
+    bottom: 0; 
+    background: rgb(51, 179, 151);
```
Comment (Teamustu) [inline news.css:23]: why are there two backgrounds?

File: news.css [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+    background: rgba(51, 179, 151, 0.5);
+    color: rgb(255,255,255);
```
Comment (Teamustu) [inline news.css:25]: as well as two color?

**Review by Teamustu — APPROVED** [review]
Looks good.

**Review by khoanguyendinh — APPROVED** [review]

**Pull request merged**